src/System.Runtime.Serialization.Xml/tests/DataContractSerializer.cs

```diff
@@ -1215,6 +1215,16 @@ public static void DCS_Nullables()
     }
 
     [Fact]
+    public static void DCS_SimpleStructWithProperties()
+    {
+        SimpleStructWithProperties x = new SimpleStructWithProperties() { Num = 1, Text = "Foo" };
+        var y = SerializeAndDeserialize(x, "<SimpleStructWithProperties xmlns=\"http://schemas.datacontract.org/2004/07/SerializationTypes\" xmlns:i=\"http://www.w3.org/2001/XMLSchema-instance\"><Num>1</Num><Text>Foo</Text></SimpleStructWithProperties>");
+
+        Assert.Equal(x.Num, y.Num);
```

Comment:
Nit: Assert.True with a helpful error message would be easier to diagnose failures. Not required.